models/stg_recharge__address_shipping_line.sql

```diff
@@ -25,8 +25,8 @@ final as (
         price,
         code,
         title,
-        _fivetran_deleted,
-        cast(_fivetran_synced as {{ dbt.type_timestamp() }}) as _fivetran_synced
+        _fivetran_deleted, 
+        cast(_fivetran_synced as {{ dbt.type_timestamp() }}) as _fivetran_synced 
```

Comment:
Will we use this field at all downstream? Typically we don't use the fivetran_synced field and we can ommit it from the staging models.

If we are going to remove, let's remove it from all the staging models.

Reply:
Removed.

Reply:
Great observation! The `loaded_at_field` is used to [calculate freshness](https://docs.getdbt.com/reference/resource-properties/freshness#definition). Since we have removed the freshness test, this is _technically_ not needed anymore. However, I think it would be good to still include as we may want to expand the freshness in a future update.

Reply:
This update looks good! I did have one additional question that I would be curious to understand. I know some models (plan for example) have both a `is_deleted` (I assume generated from Recharge themselves) and a `_fivetran_deleted` field (generated from Fivetran). Is there a possibility that these fields do not match?

I think going forward we should definitely filter on the _fivetran_deleted column, but curious if these two fields have potential to confuse customers if they have differing functions.

Reply:
I see what you mean. I have removed `_fivetran_deleted ` from the output of the staging models and kept it in the source for filtering.

Reply:
Great thanks! I do still want to know what the different between `_fivetran_deleted` and `is_deleted` are 🤔. This may be a good question for Elan.

Reply:
Looking at the [recharge docs](https://developer.rechargepayments.com/2021-01/webhooks_explained), it does appear that some of these tables use a recharge-generated deleted flag, though not all of these seem to correspond to a column in the fivetran ERD. `Plan` no long appears to have `is_deleted`, but those columns aren't finalized yet anyway. However, for `order` and `one_time_product` I think it would make sense to keep `is_deleted` in case the user wanted to filter them out?